### PR TITLE
Setup Docker config, added CORS library

### PR DESCRIPTION
**Pull request by tmclean15**
There are a few other small changes to make the Docker config compatible with  the rest of the app.

### CODE REVIEW

File: server/app.py [hunk 1]
```diff
@@ -1,19 +1,23 @@
...
-db = SQLAlchemy(app) 
+# Allows CORS on all routes (for development convenience). TODO: CORS config
+# should be changed for production.
+cors = CORS(app)
+
+db = SQLAlchemy(app)
 migrate = Migrate(app, db)
 
 # for flask-migrate to generate migration scripts
-from db_models import pluser
```
Comment (MalcolmAG) [inline server/app.py:13]: Is there a reason you removed this model import?

**Review by MalcolmAG — APPROVED** [review]
It runs well on my machine. I have the one comment I'd like clarification on, but it's still ready to be merged.

File: server/app.py (hunk 1 above)
Comment (tmclean15) [reply server/app.py:13]: It was giving me a circular import error, as the app module was importing pluser, and the pluser module was trying to import the app module. I've realized that this was an issue with VS Code's formatting bringing the import to the top of the file. I've re-added the import with a new commit.

**Review by MalcolmAG — APPROVED** [review]
Looks good

**Pull request merged**